src/main/java/com/puppycrawl/tools/checkstyle/checks/naming/IllegalIdentifierNameCheck.java

```diff
@@ -27,9 +27,9 @@
 /**
  * <p>
  * Checks identifiers with a pattern for a set of illegal names, such as those
- * that are restricted or contextual keywords. Examples include "yield", "record", "_", and
+ * that are restricted or contextual keywords. Examples include "yield", "record", and
  * "var". Please read more at
- * <a href="https://docs.oracle.com/javase/specs/jls/se14/html/jls-3.html#jls-3.9">
+ * <a href="https://docs.oracle.com/javase/specs/jls/se22/html/jls-3.html#jls-3.9">
```

Comment:
@mahfouz72 @nrmancuso What do you guys think of making this an issue? Finding all old JLS and moving them to 22.

Reply:
Good idea I am ok with this it can be a separate issue to keep tacker of all checks or I can update it in other PRs (normal updates) whenever I see an old link in the doc 
I think a separate issue would be better

Reply:
But does updating links add value? I think most of the links either 22 or 14 will have no difference in the content. there will be a difference ofc in links related to the new syntax (patterns and switch)

Reply:
@rnveach you can create an issue for this, but we will not cover it during GSOC.

Reply:
Done.